src/address.js

```diff
@@ -83,7 +97,7 @@ class Address {
     })
   }
 
-  getBalance(hash, filterOptions) {
+  getBalanceLatest(hash, filterOptions) {
```

Comment:
@taylorjdawson why `getBalanceLatest`?

I thought we were conforming to ethers & web3?